File: testData/src/java8/pkg/TestTryVar.java
package pkg;

public class TestTryVar {
  public int test(int x, int y) {
    for (int i = 0; i < 10; i++) {
      try {
        x = y + i;
        x = 5000 - i;
        x = y / y;
      } catch (Throwable t) {

      }
    }

    return x;
  }
}

File: plugins/kotlin/testData/src/kt/pkg/TestTryVar.kt
package pkg

class TestTryVar {
    fun test(x: Int, y: Int): Int {
        var x = x
        for (i in 0..9) {
            try {
                x = y + i
                x = 5000 - i
                x = y / y
            } catch (t: Throwable) {
            }
        }

        return x
    }

    fun test2(x: Int, y: Int): Int {
        var x = x
        for (i in 0..9) {
            try {
                x = y + i
                x = (5000 - i / (7 - i).let { x += it; x }) / y
            } catch (t: Throwable) {
            }
        }

        return x
    }
}
